aiounifi/models/traffic_route.py

```diff
@@ -88,21 +114,59 @@ def id(self) -> str:
         """ID of traffic route."""
         return self.raw["_id"]
 
+    @property
+    def action(self) -> str:
+        """What action is defined by this traffic route."""
+        return self.raw["action"]
+
     @property
     def description(self) -> str:
         """Description given by user to traffic route."""
         return self.raw["description"]
 
+    @property
+    def domains(self) -> list[Domain]:
+        """What IP addresses are matched against by this traffic route.
+
+        Note: This requires clients to use the UniFi network devices as their DNS server.
```

Comment:
```suggestion
        Note: This requires clients to use the UniFi network devices as their DNS server.
```
This line is a bit weird to me, with devices you mean the UniFi gateway/router right?

Reply:
I'll reword this to be the exact message from the documentation: `Note: Domain matching requires the client devices to use the UniFi gateway as the DNS server.`